src/utils/auth.js

```diff
@@ -0,0 +1,82 @@
+export const BASE_URL = 'https://api.kristinastasevich.students.nomoredomainssbs.ru';
+
+const customFetch = (url, headers) => {
```

Comment:
no need to define `customFetch` 2 times in 2 files. You can  do it once and import to any file you need